### DISCUSSION TITLE
UserService | Initializer for conditional binding must have Optional type, not 'User' error

### DISCUSSION THREAD

**Discussion by bradjeancy** [community/community#145356] [Programming Help]
### Select Topic Area

Question

### Body

Hello All,

Can someone help me with this issue I am facing whilst trying to set up a userService in Swift UI. How do I set the binding to have an optional type correctly? I tried a few other ways but still not working. Any help will be appreciated. Thanks 

[
![2024-11-23_05-21-24](https://github.com/user-attachments/assets/cad73ac8-ba23-45a8-b042-65feb5208e6c)
](url)

**Comment by shinybrightstar** [comment]
Welcome to the GitHub Community, @bradjeancy , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
